parsers/CR.py

```diff
@@ -8,115 +8,120 @@
+#     "Aeroenergía": "wind",
+#     "Altamira": "wind",
+#     "Angostura": "hydro",
+#     "Arenal": "hydro",
+#     "Balsa Inferior": "hydro",
+#     "Barranca": "oil",
+#     "Barro Morado": "geothermal",
+#     "Belén": "hydro",
+#     "Bijagua": "hydro",
+#     "Birris12": "hydro",
+#     "Birris3": "hydro",
+#     "Boca de Pozo": "hydro",
+#     "CNFL": "hydro",
+#     "Cachí": "hydro",
+#     "Campos Azules": "wind",
+#     "Canalete": "hydro",
+#     "Cariblanco": "hydro",
+#     "Carrillos": "hydro",
+#     "Caño Grande": "hydro",
+#     "Caño Grande III": "hydro",
+#     "Chiripa": "wind",
+#     "Chocosuelas": "hydro",
+#     "Chucás": "hydro",
+#     "Cote": "hydro",
+#     "Cubujuquí": "hydro",
+#     "Daniel Gutiérrez": "hydro",
+#     "Dengo": "hydro",
+#     "Don Pedro": "hydro",
+#     "Doña Julia": "hydro",
+#     "Echandi": "hydro",
+#     "Electriona": "hydro",
+#     "El Encanto": "hydro",
+#     "El Angel": "hydro",
+#     "El Angel Ampliación": "hydro",
+#     "El Embalse": "hydro",
+#     "El General": "hydro",
+#     "El Viejo": "biomass",
+#     "Garabito": "oil",
+#     "Garita": "hydro",
+#     "Guápiles": "oil",
+#     "Hidrozarcas": "hydro",
+#     "Jorge Manuel Dengo": "hydro",
+#     "La Esperanza (CoopeL)": "hydro",
+#     "La Joya": "hydro",
+#     "Las Pailas II": "geothermal",
+#     "Los Negros": "hydro",
+#     "Los Negros II": "hydro",
+#     "Los Santos": "wind",
+#     "MOVASA": "wind",
+#     "Matamoros": "hydro",
+#     "Miravalles I": "geothermal",
+#     "Miravalles II": "geothermal",
+#     "Miravalles III": "geothermal",
+#     "Miravalles V": "geothermal",
+#     "Moín I": "oil",
+#     "Moín II": "oil",
+#     "Moín III": "oil",
+#     "Orosí": "wind",
+#     "Orotina": "oil",
+#     "Otros": "hydra",
+#     "PE Cacao": "wind",
+#     "PE Mogote": "wind",
+#     "PE Río Naranjo": "hydro",
+#     "PEG": "wind",
+#     "Pailas": "geothermal",
+#     "Parque Solar Juanilama": "solar",
+#     "Parque Solar Miravalles": "solar",
+#     "Peñas Blancas": "hydro",
+#     "Pirrís": "hydro",
+#     "Plantas Eólicas": "wind",
+#     "Platanar": "hydro",
+#     "Pocosol": "hydro",
+#     "Poás I y II": "hydro",
+#     "Reventazón": "hydro",
+#     "Río Lajas": "hydro",
+#     "Río Macho": "hydro",
+#     "Río Segundo": "hydro",
+#     "San Antonio": "oil",
+#     "San Lorenzo (C)": "hydro",
+#     "Sandillal": "hydro",
+#     "Suerkata": "hydro",
+#     "Taboga": "biomass",
+#     "Tacares": "hydro",
+#     "Tejona": "wind",
+#     "Tilawind": "wind",
+#     "Torito": "hydro",
+#     "Toro I": "hydro",
+#     "Toro II": "hydro",
+#     "Toro III": "hydro",
+#     "Tuis (JASEC)": "hydro",
+#     "Valle Central": "wind",
+#     "Vara Blanca": "hydro",
+#     "Ventanas": "hydro",
+#     "Ventanas-Garita": "hydro",
+#     "Vientos de La Perla": "wind",
+#     "Vientos de Miramar": "wind",
+#     "Vientos del Este": "wind",
+#     "Volcán": "hydro",
+# }
+
+# Note: Ignoring Intercambio deliberately
```

Comment:
Is Intercambio the exchange data? If so could it be used to create a exchange parser?

Reply:
Yes Intercambio is indeed the exchange data and was categorized into North (Nicaragua) and South (Panama). Added a new exchange parser based on the already existing parser https://github.com/electricitymaps/electricitymaps-contrib/blob/master/parsers/ENTE.py. Does fetch_exchange() have to implemented in CR.py or can we just use ENTE.py as an exchange parser using the config/zones/CR.yaml config?

Reply:
There is no requirement for a fetch_exchange() method as well but having it would be nice.
I haven't checked which parser would be "best" in terms of data and quality but having a second option is always best in case the first one becomes unavailable.

Reply:
The fetch_exchange() in CR.py is now basically the same as in ENTE.py using the same data source. My question was rather about whether there has to be an exchange parser for CR in CR.py explicitly or can we just set ENTE.py to be the exchange parser for CR in its respective config file?

Reply:
Oh they are both using the same source (and domain)?
If that's the case it's is indeed pretty redundant but I assume it's already been implemented now so lets keep it. I'll try and do a full review of this PR before the weekend.

Reply:
Any update on this one?

Reply:
I have been pretty busy so didn't have time, but I have a lot of free time the rest of the week so I promise I'll get it done this week at the very least.